howso/ablation.amlg

```diff
@@ -111,23 +111,21 @@
 
 	;reduce the trained data by removing cases which have an influence weight entropy that falls above
 	; a threshold.
-	;
-	;parameters:
-	; distribute_weight_feature: optional, default !autoAblationWeightFeature. name of feature whose values to use as case weights
-	; features: default all trained features. a list of features to use when computing influence weight entropies
-	; influence_weight_entropy_threshold: optional, default !autoAblationInfluenceWeightEntropyThreshold. cases with
-	; 	influence weight entropy above this quantile will be removed
-	; skip_auto_analyze: default false. whether to skip auto analyzing as cases are removed
 	#reduce_data
 	(declare
 		(assoc
 			;{type "string"}
+			;default all trained features. a list of features to use when computing influence weight entropies
```

Comment:
Looks like the wrong description got pasted here. This is the one for `features` (the below param). Should be able to grab the description for distribute_weight_feature from main still

Reply:
Addressed in: [d54a657](https://github.com/howsoai/howso-engine/pull/239/commits/d54a6575c30a5c75415b2cc52e96d51cfb0406c6)